.github/workflows/claude.yml

```
@@ -146,101 +151,77 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Run Automatic Code Review
+      - name: Check PR Size Limits
+        id: size-check
+        run: |
+          # 変更ファイル数をチェック
+          CHANGED_FILES=$(git diff --name-only HEAD~1 | wc -l)
+          echo "changed_files=$CHANGED_FILES" >> $GITHUB_OUTPUT
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)